docs/concepts/flox-vs-containers.md

```diff
@@ -0,0 +1,436 @@
+---
+Title: Flox vs. container workflows
+Description: How does a Flox workflow differ from a container workflow?
+---
+
+# Flox vs. container workflows
+
+Containers are everywhere these days.
+They're the de-facto deployment method in the industry,
```

Comment:
**nitpick:** I'd suggest that *de facto* as a Latin phrase doesn't really need a hyphen; maybe go with italics if you want to be pretentious. Otherwise, just "de facto" will do fine.

Reply:
Don't worry about it, that's already merged